src/Models/NonhydrostaticModels/update_nonhydrostatic_model_state.jl

```diff
@@ -13,25 +13,31 @@ Update peripheral aspects of the model (halo regions, diffusivities, hydrostatic
 """
 function update_state!(model::NonhydrostaticModel)
     
+    @apply_regionally local_calculations!(model)
+
+    # Fill halos for velocities and tracers
+    fill_halo_regions!(merge(model.velocities, model.tracers),  model.clock, fields(model))
+    fill_halo_regions!(model.diffusivity_fields, model.clock, fields(model))
+    fill_halo_regions!(model.pressures.pHY′)
+
+    return nothing
+end
+
+function local_calculations!(model)
```

Comment:
```suggestion
function local_nonhydrostatic_state_update!(model)
```

even though this function appears right next to where it's used, it's possible that we encounter it in external contexts (ie in an error). So it's helpful if we give it a descriptive name (since it's not much extra effort to do that, we imght as well...)